tradelogs/storage/country_stats.go

```
@@ -49,8 +59,12 @@ func convertQueryResultToCountry(row influxModel.Row) (map[uint64]*common.Countr
 	if len(row.Values) == 0 {
 		return nil, nil
 	}
+	idxs, err := countryStatSchema.NewFieldsRegistrar(row.Columns)
+	if err != nil {
+		return nil, nil
```

why the error is ignored?